api/rbdigital.py

```diff
@@ -1940,7 +1940,11 @@ def __init__(self, _db, collection, api_class=RBDigitalAPI,
         super(RBDigitalSyncMonitor, self).__init__(_db, collection)
         self.api = api_class(_db, collection, **api_class_kwargs)
 
-    def run_once(self, start, cutoff):
+    def run_once(self, progress):
+        """Find books in the RBdigital collection that changed recently.
+
+        :param progress: A TimestampData, ignored.
```

Comment:
Why are some of these `progress` params ignored?

Reply:
It depends on the monitor. If a monitor doesn't need to know anything about the circumstances under which it was last run, it can ignore the incoming `progress`. Even in my working branch, when we need to set `achievements`, we will usually be doing this by creating a new `TimestampData` object and we might still ignore the one that's passed in to them ethod.